### ISSUE TITLE
Some libraries fail to install if node is not present on the PATH

### ISSUE THREAD

**Issue by rohanag12** [pubref/rules_node#40]
I am trying to build a simple node app in an environment without NodeJS installed.

`bazel build foo/src:bar_node_module`

Most of the dependencies get installed successfully, but one package ([chromedriver](https://github.com/giggio/node-chromedriver)) fails with this error:

```
error /home/ubuntu/.cache/bazel/_bazel_ubuntu/9a9b3273327e7f32d450222011289241/external/bar-module-deps/node_modules/chromedriver: Command failed.
Exit code: 127
Command: node install.js
Arguments: 
Directory: /home/ubuntu/.cache/bazel/_bazel_ubuntu/9a9b3273327e7f32d450222011289241/external/bar-module-deps/node_modules/chromedriver
Output:
/bin/sh: 1: node: not found
 and referenced by '//foo/src:bar_node_module'
```

I believe the issue is in chromedriver's `package.json` file:

```
...
"scripts": {
    "install": "node install.js"
  }, ...
```

Since node is not present on the path, this step fails. Is there any way to get around this?

**Comment by pcj** [comment]
Think I have a fix for this in a larger PR that will prepend the node dir to $PATH in the launcher script.  I hope to have that up by tonight.

**Comment by pcj** [comment]
Should be fixed with #41.  Pls re-open if this remains an issue @rohanag12.
